DESCRIPTION

```diff
@@ -7,18 +7,21 @@ Authors@R: c(
            role = c("aut", "cre"),
            email = "zkamvar@carpentries.org",
            comment = c(ORCID = "0000-0003-1458-7108")),
-    person(given = "François",
-           family = "Michonneau",
-           role = c("ctb"),
-           email = "francois@carpentries.org"),
     person(given = "Julien",
            family = "Colomb",
            role = c("ctb"),
            comment = c(ORCID = "0000-0002-3127-5520")),
+    person(given = "Benjamin",
+           family = "Companjen",
+           role = c("ctb")),
```

Comment:
@bencomp, I've added your contribution credit here. Feel free to make another PR if you wish for your ORCiD and/or email to appear here as well